### DISCUSSION TITLE
how to remove domain name and find pc name?

### DISCUSSION THREAD

**Discussion by myssv** [AdguardTeam/AdGuardHome#6414] [Q&A]
Hi there,

I have AdGuard Home up and running.

In the top lists I have some entries:

![grafik](https://github.com/AdguardTeam/AdGuardHome/assets/65643813/47ce289f-4b65-4916-b96c-76bdd0771a3f)

2 questions:

- How can I remove my local domain "fritz.box"?
- How can I show the real name of the IPv6 address?

Thanks for your help!

**Comment by liyiranlab** [comment]
Refactor  Top clients
![name](https://github.com/AdguardTeam/AdGuardHome/assets/58244219/acc65cd8-4c2b-4dfa-9281-576261100227)
现在的 Top clients 要显示客户端名字非常的苛刻，ipv4还好说，ipv6地址种类繁多，所以就成为上图的样子，大多数情况是一串ipv6码，非常不直观，我希望各位大神，以MAC地址为识别特征，为每个相同拥有相同MAC地址的设备加上设备名字，ipv4大多数情况可以获取设备名字，这样就可以通过拥有相同MAC地址的设备的IPv6地址加上设备名称。不知道可不可行。

**Reply by liyiranlab** [reply]
DHCPv6 只能显示一种IPv6地址的设备名，而实际IPv6会更多会使用临时IPv6地址，实际情况是无法获取设备名。希望各位大神改变思路，用MAC地址统一命名。

**Comment by shentong0722** [comment]
我用docker部署在VPS上提供公共DNS服务，之前一直无法获取到用户IP，一直显示的是容器的内部IP，今晚突然就可以获取到用户IP了，有大佬知道是什么原理吗？
![image](https://github.com/AdguardTeam/AdGuardHome/assets/63089521/fb0837bc-45a0-4008-ac7a-6db939f41415)
